Tools/Scripts/webkitpy/benchmark_runner/benchmark_runner.py

```diff
@@ -176,6 +180,8 @@ def _run_benchmark(self, count, web_root):
 
                 finally:
                     self._browser_driver.restore_env()
+                if self._pgo_profile_output_dir:
+                    shutil.copytree(self._pgo_profile_output_dir, self._diagnose_dir, dirs_exist_ok=True)
```

Comment:
Do we want to assert the existing of `self._pgo_profile_output_dir`?

Reply:
I don't think so. It's created above (L157), so if it does not exist at this point, something has gone horribly wrong and it would be appropriate to crash with a `FileNotFoundError`.